app/models/order.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 class Order < ActiveRecord::Base
   belongs_to :user
-  has_many :dishes, dependent: :destroy
+  has_many :dishes, -> { order(:name) }, dependent: :destroy
```

Comment:
As this ordering is mostly a display issue, is the association definition really the place to enforce it? Shouldn't it be done in the display layer?